### DISCUSSION TITLE
Please Add these Features:

### DISCUSSION THREAD

**Discussion by geekytricky** [TeamNewPipe/NewPipe#11062] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Please add these features:
-Channel Blocker
-Videos Grid Columns Customization (4-10)
-Add Certain Words, Phrases Filter
-Upgrade Subscriber Organizer

**Comment by wb9688** [comment]
Please create issues with the feature request template to request features at https://github.com/TeamNewPipe/NewPipe/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml (if they have not been suggested before).
